app/src/main/java/com/duckduckgo/app/settings/db/SettingsDataStore.kt

```diff
@@ -61,7 +60,7 @@ interface SettingsDataStore {
     fun clearAppBackgroundTimestamp()
 }
 
-class SettingsSharedPreferences @Inject constructor(private val context: Context) : SettingsDataStore {
+class SettingsSharedPreferences constructor(private val context: Context) : SettingsDataStore {
```

Comment:
@aitorvs I've removed the `@Inject` from this class. Let me know if it's ok or you prefer to remove the provider method.

Reply:
Tiny thing, you can remove `constructor` as well.